app/src/main/java/net/sourceforge/opencamera/PreferenceHandler.java

```diff
@@ -52,6 +52,10 @@ public boolean isSyncFormatEnabled() {
         return mSharedPreferences.getBoolean(PreferenceKeys.SyncFormatPreferenceKey, false);
     }
 
+    public boolean isTagSyncEnabled() {
```

Comment:
Can you briefly explain how this value is set when I enable the same preference in UI with switcher?

Reply:
When you click in ui, the preference of EnableTagSyncPreferenceKey changes